script/build.sh

```diff
@@ -12,15 +12,16 @@ export LC_ALL=en_US.UTF-8;
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd );
 
 export TIMESTAMP=$(date +"%m-%d-%Y-%H:%M:%S");
-export PBF2JSON_BIN="/var/www/pelias/pbf2json/build/pbf2json.linux-x64";
+export PBF2JSON_BIN="$DIR/../node_modules/pbf2json/build/pbf2json.linux-x64";
 
 # location of data files
-export POLYLINE_FILE="/data/polyline/planet.polylines"; # the file containing all the streets
-export OAPATH="/data/oa"; # base path of openaddresses file system
-export PBF2JSON_FILE="/data/pbf/planet.osm.pbf";
+export WORKINGDIR="${WORKINGDIR:-"/mnt/pelias"}"
+export POLYLINE_FILE="/$WORKINGDIR/data/polylines/planet.polylines"; # the file containing all the streets
+export OAPATH="/$WORKINGDIR/data/oa"; # base path of openaddresses file system
+export PBF2JSON_FILE="/$WORKINGDIR/data/osm/planet-latest.osm.pbf";
 
 # a directory where all builds will live
-BUILDS="/data/builds";
+BUILDS="/$WORKINGDIR/interpolation/builds";
```

Comment:
I removed all the leading slashes but I wonder what would happen if we left them. It would prevent relative paths from working, which is nice. However it would force all paths to descend from the root directory, which might cause problems. There are lots of tricky things that can happen with using variables in paths and it's really hard to think about them all.